backend/src/schema/resolvers/reports.js

```diff
@@ -58,54 +59,68 @@ export default {
     },
   },
   Query: {
-    reports: async (_parent, _params, { driver }, _resolveInfo) => {
+    reports: async (_parent, params, context, _resolveInfo) => {
+      const { driver } = context
       const session = driver.session()
-      const res = await session.run(
-        `
+      let response
+      let orderByClause
+      switch (params.orderBy) {
+        // case 'createdAt_asc':
+        //   orderByClause = 'ORDER BY report.createdAt ASC'
+        //   break
```

Comment:
I don't understand why this is commented out?
maybe because there is no UI to allow `moderators` to order how they want, but I am guessing.
If that is the case, I don't see the need to leave it commented out, since there is more than one UI... we have the playground, for example.

if there is no `orderBy` it just comes in whatever order the database happens to return it?

Reply:
Okay. @mattwr18 
I was uncertain how to handle this.

> if there is no orderBy it just comes in whatever order the database happens to return it?

Yes! That was the cause I’ve seen that no ordering was applied.